portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/elasticsearch/Elasticsearch7.testcase

```diff
@@ -64,7 +64,7 @@ definition {
 				-d '{
 					"query": {
 						"match": {
-							"_type": "LiferayDocumentType"
+							"_type": "_doc"
```

Comment:
@xbrianlee let me know if you think there are any issues related to this test case. if there are issues i think we can probably just change it from a match query to a match_all query and not have to deal with the type